pkg/systemd/services/services.jsx

```diff
@@ -926,19 +956,16 @@ const ServicesPageFilters = ({
         onDeleteChip();
     }, [setCurrentTextFilter, onDeleteChip]);
 
+    const onTextFilterChanged = textFilter => {
+        setCurrentTextFilter(textFilter);
+        cockpit.location.go([], ({ ...options, name: textFilter }));
+    };
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18115-20230103-122827-2f1a70ab-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)